<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_cs</class>
 <widget class="QDialog" name="Dialog_cs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_close">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>370</y>
     <width>103</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_gen">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>240</y>
     <width>103</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>341</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bitstream Charter</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Affine Cipher</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>441</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>This module generates the 'training data' for cryptanalysis</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>171</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Enter plain-text below</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>491</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Training set shall be appended at: &quot;&lt;project_root&gt;/data/TSet.dat&quot; </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>300</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btn_close</sender>
   <signal>clicked()</signal>
   <receiver>Dialog_cs</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>378</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>350</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
